src/main/java/com/sarality/db/modifier/IgnoreEmptyValueColumnUpdater.java

```diff
@@ -0,0 +1,47 @@
+package com.sarality.db.modifier;
+
+import android.content.ContentValues;
+import android.text.TextUtils;
+
+import com.sarality.db.Column;
+import com.sarality.db.RecordModifier;
+import com.sarality.db.content.ContentValueWriter;
+import com.sarality.db.io.ColumnValueReader;
+
+/**
+ * When updating a row, ignore any updates to a column when the value is null or empty
```

Comment:
Sorry, but how are we going to use this? If we initialize this table definition, how will we update the column when we perform the sync from the cloud down to the app?